generators/client/templates/react/src/main/webapp/app/reducers/account.ts

```diff
@@ -57,17 +57,17 @@ const apiUrl = '/api/account';
 export const saveAccountSettings: ICrudPutAction = account => ({
   type: ACTION_TYPES.UPDATE_ACCOUNT,
   meta: {
-    successMessage: messages.DATA_CREATE_SUCCESS_ALERT,
-    errorMessage: messages.DATA_UPDATE_ERROR_ALERT
+    successMessage: translate('settings.messages.success'),
+    errorMessage: messages.DATA_ERROR_ALERT
   },
   payload: axios.post(apiUrl, account)
 });
 
 export const savePassword: ICrudPutAction = password => ({
   type: ACTION_TYPES.UPDATE_PASSWORD,
   meta: {
-    successMessage: messages.DATA_CREATE_SUCCESS_ALERT,
-    errorMessage: messages.DATA_UPDATE_ERROR_ALERT
+    successMessage: translate('password.messages.success'),
+    errorMessage: translate('password.messages.error')
```

Comment:
Here we have a specific message so we can override by setting here, so this is fine